pkg/platform/model/auth.go

```diff
@@ -34,25 +34,24 @@ func WaitForAuthorization(deviceCodePayload *mono_models.DeviceCode) (*mono_mode
 	deviceCode := strfmt.UUID(*deviceCodePayload.DeviceCode)
 	getParams := oauth.NewAuthDeviceGetParams()
 	getParams.SetDeviceCode(deviceCode)
-	startTime := time.Now()
-	const timeout = 5 * 60 * time.Second
```

Comment:
Did you remove the timeout altogether?

Reply:
Yep: https://github.com/ActiveState/cli/pull/1716/commits/25d226c4a638aa4a2e4e314ce933830852af46d4

Reply:
Won't this lead to polling indefinitely if the user doesn't complete the auth?

Reply:
Gah, my bad. I linked the wrong commit: https://github.com/ActiveState/cli/pull/1716/commits/2204dbc4e5f197762ee6fa73d52f9755cf48e8e3

The Platform has its own timeout and we stop polling when that timeout is reached.